dds_web/database/models.py

```diff
@@ -280,8 +285,9 @@ class User(flask_login.UserMixin, db.Model):
     username = db.Column(db.String(50), primary_key=True, autoincrement=False)
     name = db.Column(db.String(255), unique=False, nullable=True)
     _password_hash = db.Column(db.String(98), unique=False, nullable=False)
-    _otp_secret = db.Column(db.String(32))
-    has_2fa = db.Column(db.Boolean)
+    hotp_secret = db.Column(db.LargeBinary(20), unique=False, nullable=False)
+    hotp_counter = db.Column(db.BigInteger, unique=False, nullable=False, default=0)
+    hotp_requested_time = db.Column(db.DateTime, unique=False, nullable=True)
```

Comment:
could we change this to `hotp_issue_time` ?

Reply:
That's done by your PR already.